### PR TITLE
Fix FP division by zero (issue #3995)

### PR DESCRIPTION
**Pull request by stweil**

### CODE REVIEW

File: src/wordrec/language_model.cpp [hunk 1]
```diff
@@ -1375,7 +1375,12 @@ void LanguageModel::ExtractFeaturesFromPath(const ViterbiStateEntry &vse, float
   // features[PTRAIN_NUM_BAD_FONT] = vse.consistency_info.inconsistent_font;
 
   // Classifier-related features.
-  features[PTRAIN_RATING_PER_CHAR] = vse.ratings_sum / static_cast<float>(vse.outline_length);
+  if (vse.outline_length != 0.0f) {
```
Comment (zvezdochiot) [inline src/wordrec/language_model.cpp:1378]:
```c
features[PTRAIN_RATING_PER_CHAR] = (vse.outline_length > 0.0f) ? (vse.ratings_sum / static_cast<float>(vse.outline_length)) : 0.0f;
```

Comment (stweil) [reply src/wordrec/language_model.cpp:1378]: Why would that be better?

Comment (zvezdochiot) [reply src/wordrec/language_model.cpp:1378]: Hi @stweil .

:warning: `(vse.outline_length != 0.0f)`!!!

Comment (stweil) [reply src/wordrec/language_model.cpp]: Thanks. I replaced `!=` by `>` now. Please review the updated code.

Comment (zvezdochiot) [reply src/wordrec/language_model.cpp]: Hi @stweil .

`vse.outline_length` it's `(float)`?

**Review by zvezdochiot — APPROVED** [review]

**Pull request merged**